### PR TITLE
Feec dirichlet

### PR DESCRIPTION
**Pull request by saidctb**
The following changes were made:
  - Delete node annotation
  - Compute boundary jacobian when dim = 1
  - Update library version

### CODE REVIEW

File: sympde/topology/mapping.py [hunk 1]
```diff
@@ -1065,18 +1061,24 @@ def eval(cls, expr, mapping=None, dim=None, **options):
...
 
         elif isinstance(expr, LinearForm):
-            tests  = [get_logical_test_function(a) for a in expr.test_functions]
-            body   = cls.eval(expr.expr, mapping=mapping, dim=dim)
+            tests   = [get_logical_test_function(a) for a in expr.test_functions]
+            mapping = expr.domain.mapping
+            dim     = expr.domain.dim
+            body    = cls.eval(expr.expr, mapping=mapping, dim=dim)
             return LinearForm(tests, body)
         elif isinstance(expr, Norm):
```
Comment (yguclu) [inline sympde/topology/mapping.py:1077]: Can you please add an empty line between these two lines?

File: sympde/expr/evaluation.py [hunk 2]
```diff
@@ -480,8 +459,10 @@ def eval(cls, *_args, **kwargs):
             J    = expr.mapping.jacobian_expr
             if axis is None:
                 return J
-            else:
+            elif expr.mapping.ldim>1:
```
Comment (yguclu) [inline sympde/expr/evaluation.py:462]: Can you please add empty spaces around the comparison operator?

**Review by yguclu — CHANGES_REQUESTED** [review]
Very good! Can you please fix a couple of formatting issues?

**Comment by yguclu** [comment]
If you want to upload a new version of SymPDE to PyPI, you need to update the version number.

File: sympde/topology/mapping.py (hunk 1 above)
Comment (saidctb) [reply sympde/topology/mapping.py]: done

File: sympde/expr/evaluation.py (hunk 2 above)
Comment (saidctb) [reply sympde/expr/evaluation.py]: done

**Comment by saidctb** [comment]
> If you want to upload a new version of SymPDE to PyPI, you need to update the version number.

done

**Review by yguclu — APPROVED** [review]

**Pull request merged**